www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -1,6 +1,6 @@
 <model>
   <mount>//OPNsense/Nginx</mount>
-  <version>1.20.1</version>
+  <version>1.24.0</version>
```

Comment:
just as a comment the model version really doesn't have to correspond to the plugin version, but doesn't matter either way